server/tilesource/tiff_reader.py

```diff
@@ -139,7 +139,7 @@ def _validate(self):
 
         :raises: ValidationTiffException
         """
-        if self._tiffFile.GetField('SamplesPerPixel') != 3:
+        if self._tiffFile.GetField('SamplesPerPixel') < 3:
```

Comment:
The python-tests branch uses a png which results in 4 channel image.  When converted by vips, the TIF still reports four channels, but the JPEGs served for the tiles are all 3 channel.  This is the use case I really want to cover.

JPEGs can by more than 3 channel without the fourth channel being alpha (CMYK is common).  I've also seen (years ago) hyper-spectral JPEGs where 3 channels are YCbCr and additional channels are each luminance in different spectral bands.